CMakeLists.txt

```diff
@@ -73,6 +73,14 @@ set(CUSTOM_COMPILE_OPTIONS
     ""
     CACHE STRING "Allows adding custom C/C++ flags"
     )
+set(WUI
+    "YES"
+    CACHE BOOL "Enable Web User Interface"
+    )
```

Comment:
Why not follow pattern PRINTERS_WITH_... list?

Reply:
when WUI is fully functional then GUI is not necessary, eg. in farms. I see it is good not to use gui even in MINI

Reply:
there is no such request so far, use PRINTES_WITH_* list